app/src/main/java/org/oppia/android/app/ongoingtopiclist/OngoingTopicItemViewModel.kt

```diff
@@ -27,7 +28,8 @@ class OngoingTopicItemViewModel(
   }
 
   fun onTopicItemClicked() {
-    routeToTopic(internalProfileId, topic.topicId)
+    val profileId = ProfileId.newBuilder().apply { internalId = internalProfileId }.build()
```

Comment:
Ditto here--can we avoid creating a new ProfileId?

Reply:
Updated